README.txt

```diff
@@ -1,34 +1,32 @@
 === Algolia Search ===
 Contributors: Algolia
-Tags: Search, Algolia, Autocomplete, instant-search, relevant search, search highlight, as-you-type search, search by tag, suggest, search by category, ajax search, better search, custom search
+Tags: Search, Algolia, Autocomplete, instant-search, relevant search, search highlight, faceted search, find-as-you-type search, search by tag, suggest, search by category, ajax search, better search, custom search
```

Comment:
We have to limit the tags to 12 items according to: https://developer.wordpress.org/plugins/wordpress-org/detailed-plugin-guidelines/